plugin/trino-loki/src/main/java/io/trino/plugin/loki/QueryRangeTableFunction.java

```diff
@@ -89,6 +95,11 @@ public TableFunctionAnalysis analyze(ConnectorSession session, ConnectorTransact
         LongTimestampWithTimeZone startArgument = (LongTimestampWithTimeZone) ((ScalarArgument) arguments.get("START")).getValue();
         LongTimestampWithTimeZone endArgument = (LongTimestampWithTimeZone) ((ScalarArgument) arguments.get("END")).getValue();
 
+        Long step = (Long) ((ScalarArgument) arguments.get("STEP")).getValue();
+        if (step == null) {
+            step = 0L;
+        }
```

Comment:
What happens if we specify a negative value in step argument? Please add a test.